### PR TITLE
Raises beam rifle combat requirement from 5 to 7, materials from 6 to 7

### PR DESCRIPTION
**Pull request by silicons**
i'm so done

### CODE REVIEW

**Comment by evsey9** [comment]
How much harder would it actually be to make? What would you need to get to make the rifle now?

**Comment by GunHog** [comment]
So when may we have the accelerator laser back? This pushes your rifle to a much higher status of rarity. It is quite displeasing to lose that weapon in favor of one that feels quite broken to use, one that I have to rest in order to disable the zoom mode, and one that is often severely limiting to my own sight when using.

I would like to request a revert of this weapon, please. It is not a suitable replacement for what we lost.

**Comment by kevinz000** [comment]
@GunHog zoom issues will be addressed very soon. 
also you know rnd can get combat 7 without cargo now..?

**Comment by LaharlMontogmmery** [comment]
@kevinz000 I'd love to know how.

**Comment by kevinz000** [comment]
@vcordie

**Comment by kevinz000** [comment]
@LaharlMontogmmery someone "stealthbuffed" it when they added a certain mech weapon...

**Comment by GunHog** [comment]
So, are you saying that this weapon is not as hard to get as I think it is, so you proceed to mention a method, thus eventually resulting in that method being removed...that would effectively validate my argument.

Look, I do not supporting replacing our mid-tier stuff with more powerful and expensive items! It leaves a void in that spot. I did not support your original PR because I was worried that the weapon was OP and would be made more expensive and harder to get while also being nerfed.  If this weapon were simply added to the list, then I could safely ignore it and move on. Sadly, you replaced something for this instead.

The result? A net loss. The Accel Laser was something I would build during an emergency when Cargo/Botany failed the station. The laser also had the advantage of not requiring uranium - I often faced the scenario where miners brought back diamond, but not any uranium. Now, that is gone, replaced by a rifle I will probably not see often and does not even work very well.

Please do not take this personally. It has been my personal experience with the weapon, and I am very disappointed in the outcome. I would like the Accel Laser back, it is superior when considering its original cost and simpler mechanic.

**Comment by kevinz000** [comment]
@GunHog cheridan said they would approve this if I replaced accel laser cannons with it
This is meant to be a long ranged sniping weapon that gains power far away because it'll be very hard to dodge it without an open space.
I could try buffing the angle penalties more but I doubt anything will happen.
Anyways next up is automatic zooming and maybe some more balance changes, and we can try reverting/replacing if nothing works.
I'd rather give this a chance, though.

**Comment by GunHog** [comment]
That does nothing to address my point about replacing moderate cost mid-tier items with really expensive high-tier ones. I do not like this trend, because it is robbing us of a decent weapon. It is not a good replacement because of this. We already have a good high tier, long range weapon. I AM NOT SUGGESTING THAT YOU REPLACE IT, but I am saying the 'long range weapon that gets stronger' role is already filled, and there is already a different reliable weapon at the high-tier that I would much rather carry than fumble about with using properly. I have trouble using the flight suit's zoom mode as well. Zooming is just too buggy and unreliable as a core mechanic. At least with the flight suit, I am not required to use it. I am _forced_ to do it with this rifle.

You are going to leave the mid-tier empty. You are not going to give us any replacement for the mid-tier you are taking. This is the greatest reason why I oppose this weapon, and it does not seem that this concern will be properly addressed.

**Comment by Kromgar** [comment]
All you need is a combat shotgun

**Comment by XDTM** [comment]
I feel like the midtier should be something else anyway, still laser/energy based, but experimental variations that would be sidegrades to sec weaponry.

**Comment by kevinz000** [comment]
accelerator lasers struck me as "This is a laser gun, only it's better". Yes, it's strong at long range. Too strong. It's either "You dodge it at long range with your advantage of being at long range" or "You get hit and you instantly die from getting hit with 100 burn". A different mid range weapon can be added later on but a true long ranged weapon shouldn't be "Laser gun but better". The AEG already fulfils that. 
Basically, I don't like the accel laser because of that, and I feel this is better.
@Kromgar Yes people did, it instakills blobs if you aren't trash.

**Comment by kevinz000** [comment]
(this gun doesn't suck against blobs either)

**Comment by kevinz000** [comment]
also the hard line is rnd can only have so many guns, why do you think i removed accel lasers from rnd if i could add this and keep both?

**Comment by Iamgoofball** [comment]
# hardlight bows

**Comment by Iamgoofball** [comment]
thanks for rebalancing it to 3 minutes extra of ui clicking added on top of the first 10 minutes of ui clicking this will fix the balance

**Comment by kevinz000** [comment]
You're welcome.

**Comment by sawrge** [comment]
remove r&d guns

**Comment by sawrge** [comment]
name one situation where you'd use this shit

**Comment by kevinz000** [comment]
@sawrge guess you'd rather have laser gun 2.0 then which you will if I fail to make this useful.

**Comment by sawrge** [comment]
I'd rather have neither

**Comment by Steelpoint** [comment]
Honestly I think only the highest tier of RnD weapons should outclass whatever the station starts with, I'm fine with this.

**Comment by kevinz000** [comment]
@Steelpoint this is getting rebalanced after the fixes/tweaks currently open get in, so stay tuned!
